.github/actions/install-oppia-dependencies/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Install Dependencies'
+description: 'Installs wheel and third-party python libs.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup python by installing wheel
+      run: pip install wheel==0.35.0
+      shell: bash
+    - name: Check Yarn Cache
+      run: |
+        pwd
+        cd ..
+        pwd
```

Comment:
Any specific reason for logging the pwd value?

Reply:
I log this to help make clear what folder we are moving from and into. This should help avoid confusions like you encountered in the comment below about which `.yarnrc` file we are working with

Reply:
I'll add comments instead

Reply:
done